site/Gemfile.lock

```diff
@@ -1,252 +1,102 @@
+GIT
+  remote: https://github.com/jekyll/github-metadata.git
+  revision: 9353743dc9ad6de2f522923d57eb8eb00f87abf6
+  specs:
+    jekyll-github-metadata (2.12.1)
+      jekyll (>= 3.4, < 5.0)
+      octokit (~> 4.0, != 4.4.0)
+
```

Comment:
We can't include this as a regular gem yet, is that correct? What's the course of action when the gem is available with support for Jekyll 4?

Reply:
That's correct, the Gemfile will be updated when a gem is released which supports Jekyll 4 and PR .